### PR TITLE
es upgrade

### PR DESCRIPTION
**Pull request by depsiatwal**

### CODE REVIEW

**Review by ejayesh — APPROVED** [review]
lgtm

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
           POSTGRES_PASSWORD: debug
           POSTGRES_DB: directory_api_debug
       - image: circleci/redis:3.2-alpine
-      - image: docker.elastic.co/elasticsearch/elasticsearch:6.7.1
+      - image: docker.elastic.co/elasticsearch/elasticsearch:7.1.1
```
Comment (agiamas) [inline .circleci/config.yml:14]: can we use 7.9.3 for alignment with latest stable across repos?
https://github.com/uktrade/export-opportunities/blob/706b2da7b524ab07d8303592995585b8fa957292/.circleci/config.yml#L50

**Review by agiamas — CHANGES_REQUESTED** [review]
minor change

**Review by agiamas — APPROVED** [review]
lgtm

**Pull request merged**